workflows/SetParameter.cwl

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.2
+class: Workflow
+
+label: SetParameter
+doc: |-
+    Receive, assert, and validate new simulation model parameters.
+    Update simulation model data base after passing all outlined steps.
+
+inputs:
+
+  - id: input_parameter
+    type: File
+    doc: |-
+      Updated simulation model parameter(s)
```

Comment:
No, just mentioning here since it popped into my head and we might want to decide now and change the wording accordingly. Unless you want to have separate workflows, one for adding a new version, one for adding a new parameter all together and one for updating an existing version (should be extra careful with this one).
Either way, not something to worry about now so you can resolve this now and we can come back to it when it's actually important.